content/reference-rippled.md

```diff
@@ -3012,7 +3012,7 @@ The response follows the [standard format](#response-formatting), with a success
 | account\_id | String | The [Address][] of the account. |
 | public\_key | String | The public key of the account, in encoded string format. |
 | public\_key\_hex | String | The public key of the account, in hex format. |
-{#| warning | String | (May be omitted) If the request specified a seed value, this field provides a warning that it may be insecure. TODO: uncomment when this gets added in 0.32.0 |#}
+| warning | String | (May be omitted) If the request specified a seed value, this field provides a warning that it may be insecure. _(New in [version 0.32.0][])_ |
```

Comment:
This looks fine, but noticed a few other, minor things in the rest of this section: 
- "The wallet_propose request is an admin command that cannot be run by unpriviledged users!"
    1) "unpriviledged" --> "unprivileged" 2) is this the way the other admin commands are identified? Seems fine, just thinking the format should be uniform 3) "admin command" links to "Connecting to rippled," which has a link to all the Admin commands. Should there also be a reciprocal link from the "List of Admin Commands" to "Connecting to rippled"?
- "The commandline version of this command cannot generate Ed25519 keys."
    Shouldn't this be in the description of the `key_type` parameter?
- "Specifying a Seed"
    Should this be a header? Also, it seems a little weird to start a section with a warning. Maybe the text in the "Caution" callout could just be text under this header? (Oh, it looks like it *is* level 5 header, it just renders as paragraph text. I guess our styles don't handle 5th level headers, but maybe it should.)